developers.adoc

```diff
@@ -249,6 +249,6 @@ After addition, don't forget to update lists of languages:
 * https://github.com/kaitai-io/kaitai_struct — project description
 * https://github.com/kaitai-io/kaitai_struct_compiler — project description
 * https://github.com/kaitai-io/kaitai_struct_compiler/blob/master/README.md — `-t` option documentation
-* http://kaitai.io — everywhere
+* https://kaitai.io — everywhere
```

Comment:
Any particular reason why this is explicitly https, unlike all the others?

Reply:
The reason is "not to confuse users" by showing different URL than the link points to. I just don't want to explicitly show `https://kaitai.io/` when the link actually leads to a http:// version (if we're currently at **http**://doc.kaitai.io/).

Maybe the sensible solution would be to show just `kaitai.io`:

```asciidoc
* link://kaitai.io[kaitai.io] — everywhere
```

but I'm not sure if it looks good among these `https://...` links:

> * https://github.com/kaitai-io/kaitai_struct — project description
> * https://github.com/kaitai-io/kaitai_struct_compiler/blob/master/README.md — `-t` option documentation
> * [kaitai.io](//kaitai.io/) — everywhere
> * https://bintray.com/kaitai-io/debian/kaitai-struct-compiler/view — package description
> * https://twitter.com/kaitai_io — profile

Maybe drop the https:// prefixes, as Chrome already does in the address bar?

> * [github.com/kaitai-io/kaitai_struct](https://github.com/kaitai-io/kaitai_struct) — project description
> * [github.com/kaitai-io/kaitai_struct_compiler/blob/master/README.md](https://github.com/kaitai-io/kaitai_struct_compiler/blob/master/README.md) — `-t` option documentation
> * [kaitai.io](//kaitai.io/) — everywhere
> * [bintray.com/kaitai-io/debian/kaitai-struct-compiler/view](https://bintray.com/kaitai-io/debian/kaitai-struct-compiler/view) — package description
> * [twitter.com/kaitai_io](https://twitter.com/kaitai_io) — profile

Hm, this would bloat the markup quite a bit though:

```asciidoctor
https://bintray.com/kaitai-io/debian/kaitai-struct-compiler/view[bintray.com/kaitai-io/debian/kaitai-struct-compiler/view] — package description
```

vs.

```asciidoctor
https://bintray.com/kaitai-io/debian/kaitai-struct-compiler/view — package description
```

Reply:
Maybe I'm just bikeshedding here, but as a user I'd be confused if the link that claims to lead on **https**:// version would take me to the **http**:// one.

The opposite is better: many sites are already redirecting automatically from **http**:// to **https**://, so a link that _does the same_ can't be to blame I guess.

Yeah, I'll claim it's `http://kaitai.io/` and will redirect to **http**:// or **https**:// according to the current protocol, because I think I'm just wasting time here.

Reply:
Ah, I overlooked that here the URL is also the user-visible link text. In that case it's probably fine either way. 🤷